README.md

```diff
@@ -587,75 +587,90 @@ While there are plenty of domain-name providers out there, we are going to use a
-
-
-#### VPS: Webserver NGINX
-Uvicorn is working fine, but we'll add a more robust solution, to be able to do some caching and better log-management: nginx (engine-x). We'll add a new configuration file for your website.
-
-_Please don't forget to adjust domain names and paths below accordingly_
-
-   - [ ] `sudo nano /etc/nginx/sites-available/paymeinsats.conf` to create and edit your new configuration file nginx will use
-
-Add the following entries
-```
-server {
-        # Binds the TCP port 80
-        listen 80;
-        # Defines the domain or subdomain name
-        server_name paymeinsats.duckdns.org;
-        # Redirect the traffic to the corresponding 
-        # HTTPS server block with status code 301
-        return 301 https://$host$request_uri;
-       }
-
-server {
-        listen 443 ssl; # tell nginx to listen on port 443 for SSL connections
-        server_name paymeinsats.duckdns.org; # tell nginx the expected domain for requests
-
-        access_log /var/log/nginx/paymeinsats-access.log; # Your first go-to for troubleshooting
-        error_log /var/log/nginx/paymeinsats-error.log; # Same as above
-
-        location / {
-                proxy_pass http://127.0.0.1:5000; # This is your uvicorn LNbits local host IP and port
-                proxy_set_header Upgrade $http_upgrade;
-                proxy_set_header Connection 'upgrade';
-                proxy_set_header X-Forwarded-Proto https;
-                proxy_set_header Host $host;
-                proxy_http_version 1.1; # headers to ensure replies are coming back and forth through your domain
-       }
-
-        ssl_certificate /etc/letsencrypt/live/paymeinsats.duckdns.org/fullchain.pem; # Point to the fullchain.pem from Certbot 
-        ssl_certificate_key /etc/letsencrypt/live/paymeinsats.duckdns.org/privkey.pem; # Point to the private key from Certbot
+$ sudo apt install caddy
+```
+
+##### Create the Caddyfile
+```
+$ sudo caddy stop
+$ sudo nano /etc/caddy/Caddyfile
+```
+
+##### Fill the file with an adjusted domain address
+```
+paymeinsats.duckdns.org {
+  handle /api/v1/payments/sse* {
+    reverse_proxy 0.0.0.0:5000 {
+      header_up X-Forwarded-Host paymeinsats.duckdns.org
+      transport http {
+         keepalive off
+         compression off
+      }
+    }
+  }
+  reverse_proxy 0.0.0.0:5000 {
+    header_up X-Forwarded-Host paymeinsats.duckdns.org
+  }
 }
 ```
-`CTRL-X` => `Yes` => `Enter` to save
-Next we'll test the configuration and enable it by creating a symlink from sites-available to sites-enabled.
+-> CTRL+x -> y -> RETURN
+
+##### Add caddy autostart service
+```
+$ sudo nano /etc/systemd/system/caddy.service
+```
+##### Replace content with
+```
+# caddy.service
+
+[Unit]
+Description=Caddy
+Documentation=https://caddyserver.com/docs/
+After=network.target network-online.target
+Requires=network-online.target
+
+[Service]
+Type=notify
+User=caddy
+Group=caddy
```

Comment:
Is that supposed to run as user `caddy`? If so, do we need to add this user first?

@AxelHamburch